### PR TITLE
feat(fdi-dotstatsuite): Fdi dotstatsuite logging config, upgrade chart to 0.2.15

### PR DESCRIPTION
**Pull request by GoodTown**
added logging configuration to fdi-dotstatuite, can now change log4net.config from values.yaml

### CODE REVIEW

**Comment by GoodTown** [comment]
deleted trailling spaces, ready to merge

**Comment by GoodTown** [comment]
Removed trailling spaces again

File: stable/fdi-dotstatsuite/templates/cm/nsidesign-log4net.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{.Values.nsidesignlog4netconfig.name}}
+data:  
+  log4net.config: {{ .Values.nsidesignlog4netconfig.data | indent 4 }}
```
Comment (glavers95) [inline stable/fdi-dotstatsuite/templates/cm/nsidesign-log4net.yaml:6]: GitHub flags "No newline at end of file" here. You will want to add a new line (empty line) to the end-of-file here

File: stable/fdi-dotstatsuite/templates/cm/nsistaging-log4net.yaml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ .Values.nsistaginglog4netconfig.name}}
+data:  
+  log4net.config: {{ .Values.nsistaginglog4netconfig.data | indent 4}}
```
Comment (glavers95) [inline stable/fdi-dotstatsuite/templates/cm/nsistaging-log4net.yaml:6]: Same here - GitHub flags "No newline at end of file" here. You will want to add a new line (empty line) to the end-of-file here

File: stable/fdi-dotstatsuite/templates/secret/sdmx-log-secret.yaml [hunk 3]
```diff
@@ -0,0 +1,10 @@
+{{- if or .Values.nsiStaging.sdmx_logs.enabled .Values.nsiReset.sdmx_logs.enabled}}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ .Values.fullnameOverride }}-sdmx-logs-secret
+type: Opaque
+data:
+  azurestorageaccountname: {{ .Values.managed.sdmx_log.azurestorageaccountname | b64enc }}  
+  azurestorageaccountkey: {{ .Values.managed.sdmx_log.azurestorageaccountkey | b64enc }}
+{{- end }}
```
Comment (glavers95) [inline stable/fdi-dotstatsuite/templates/secret/sdmx-log-secret.yaml:10]: GitHub flags "No newline at end of file" here. You will want to add a new line (empty line) to the end-of-file here

**Comment by glavers95** [comment]
@GoodTown  left a couple comments regarding "No newline at end of file". 

The rest of this looks great! This LGTM once comments have all been resolved. Once ready, I'll give the Approval and this will be merged in!

**Comment by GoodTown** [comment]
> @GoodTown left a couple comments regarding "No newline at end of file".
> 
> The rest of this looks great! This LGTM once comments have all been resolved. Once ready, I'll give the Approval and this will be merged in!

Empty lines added! 

perfect thank you.

**Review by glavers95 — APPROVED** [review]
This LGTM!

**Pull request merged**